### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by vitaliykravchyk**
_Complete the form below and remove this line._

### HTML-CSS-Popup

[Demo](https://vitaliykravchyk.github.io/popup) |
[Code base](https://github.com/vitaliykravchyk/frontend-2019-homeworks/tree/popup/submissions/vitaliykravchyk/popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/vitaliykravchyk/popup/index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  <header>
+    <ul class="app-list">
+      <li class="mail">
+        <a href="gmail">Gmail</a>
+      </li>
+      <li class="images">
+        <a href="images">Images</a>
+      </li>
+      <li class="popup">
+        <input type="checkbox" name="popup__checkbox" id="popup__checkbox" class="popup__checkbox"  >
```
Comment (A-Ostrovnyy) [inline submissions/vitaliykravchyk/popup/index.html]: Better use `label` for `input` for better accessibility.

**Comment by A-Ostrovnyy** [comment]
Good work!
See my comments above.

**Comment by vitaliykravchyk** [comment]
Thnx for review. Comments are fixed.

**Comment by A-Ostrovnyy** [comment]
Well Done!

**Pull request merged**